Generator/Sources/NeedleFramework/Parsing/Pluginized/Tasks/PluginizedASTDeclarationParserTask.swift

```diff
@@ -45,7 +46,7 @@ class PluginizedDeclarationsParserTask: AbstractTask<PluginizedDependencyGraphNo
         let nonCoreComponents = visitor.nonCoreComponents
         let pluginExtensions = visitor.pluginExtensions
         
-        return PluginizedDependencyGraphNode(pluginizedComponents: pluginizedComponents, nonCoreComponents: nonCoreComponents, pluginExtensions: pluginExtensions, components: baseNode.components, dependencies: baseNode.dependencies, imports: baseNode.imports)
+        return PluginizedDependencyGraphNode(pluginizedComponents: pluginizedComponents, nonCoreComponents: nonCoreComponents, pluginExtensions: pluginExtensions, components: baseNode.components, dependencies: baseNode.dependencies, imports:  Array(OrderedSet(baseNode.imports + visitor.imports)))
```

Comment:
isn't this slow? why not sort/remove dupes at the end when needed?

Reply:
Updated! These actually get stored into a Set eventually so I can just keep the array concatenation here.